### ISSUE TITLE
Checkboxes should be able to be activated by the Spacebar key instead of the Enter key

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#4617]
[Clarity Version]
3.1.1

[Test URL]
https://clarity.design/documentation/tree-view#checkbox-tree

[WCAG 2.1 Guideline]
2.1.1

[OS]
MacOS 10.15.3, Windows 10

[User Agents/Assistive Technologies]
Safari 13.0.5, Firefox 75.0

[Component]
Tree View

[Description]
Checkboxes should be able to be activated by the Spacebar key instead of the Enter key. This is in multiple examples. 

[User Impact
Typically, switch users can activate a checkbox via the Spacebar key. Instead the Spacebar key scrolls the screen and the Enter key must be used to activate the checkbox. 

[Steps to Reproduce]
See the following instructions for switch usage then follow the steps below.

Windows (see Scan through keys)
https://support.microsoft.com/en-us/help/10762/windows-use-on-screen-keyboard

Mac
https://support.apple.com/en-us/HT202865

-  Using a switch, navigate to and activate a checkbox with the Spacebar key. Notice that the screen scrolls instead of selecting the checkbox.
-  Navigate to and activate a checkbox with the Enter key

[Recommendation]
Only the spacebar should check a checkbox as seen at https://labs.levelaccess.com/index.php/Input_Element_Type_Checkbox

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
